src/main/java/com/google/api/gax/core/FlowControlSettings.java

```diff
@@ -49,12 +49,23 @@ public static FlowControlSettings getDefaultInstance() {
   @Nullable
   public abstract Integer getMaxOutstandingRequestBytes();
 
+  /** Is flow control enabled. Defaults to true. */
+  public abstract boolean getIsEnabled();
```

Comment:
It seems pretty clear to me - "enabled" is the first level of control, everything else is only used if "enabled" is true. It does make sense to have this documented to make it clearer.